skins/wmde19/src/components/layout/Footer.vue

```diff
@@ -1,11 +1,81 @@
 <template>
-	<div class="footer">
-		Footer
+	<div class="footer has-background-footer">
+		<div class="container">
+			<div class="columns">
+				<div class="column is-two-thirds">
+					<div class="level">
+						<div class="level-left">
+							<div class="logo-footer">
+								<a href="https://www.wikimedia.de/">
+									<img :src="imagePath + '/images/logo-vertical-wikimedia.svg'" alt="Wikimedia Deutschland">
+								</a>
+							</div>
+						</div>
+						<div class="level-item">
+							<div class="bank-data-content">
+								<p><strong>Kontoinhaber:</strong> Wikimedia Foerdergesellschaft</p>
+								<p>
+									<strong>{{ $t( 'iban')  }}:</strong><span class="spacer" v-for="ibanPartial in iban">{{ ibanPartial }}</span>
+								</p>
+								<p><strong>{{ $t( 'bic')  }}:</strong> {{ $t( 'operator_account_bic')  }}</p>
+								<p>Bank für Sozialwirtschaft</p>
+								<p><strong>{{ $t( 'account_number')  }}:</strong> {{ $t( 'operator_account_number')  }}</p>
+								<p><strong>{{ $t( 'blz')  }}:</strong> {{ $t( 'operator_account_blz')  }}</p>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class="column is-one-third">
+					<ul class="list-menu">
+						<li v-for="link in footerMenu">
+							<a :href="link.url">
+								<span>{{ $t( 'footer_menu_' + link.id ) }}</span>
+							</a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
 	</div>
 </template>
 
 <script>
 export default {
 	name: 'Footer',
+	props: [ 'imagePath' ],
+	computed: {
+		iban: function () {
+			return this.$t( 'operator_account_iban' ).split( ' ' );
+		},
+	},
+	data: function () {
```

Comment:
Since we committed to TypeScript we'll have to define an interface for the object returned by `data`.
I propose we have a separate folder for typescript interfaces 'cause there will be a lot of them.
I know Gabriel is not a fan of `\types` so probably `models` or `datamodels` or something will be better.
Since the layout components are simple we can have the `ts` interfaces in the same file, something like `Layout.ts`